python/src/aac/validate/_validate.py

```diff
@@ -78,6 +78,8 @@ def _with_validation(
         result = _validate_definitions(user_definitions, language_context, validate_context)
 
         if result.is_valid():
+            for definition in user_definitions:
+                print(f"The {definition.name} definition in {definition.source.uri} has been validated.")
```

Comment:
While I am fairly confident this gets us the information we wanted with this task, I am not sure that just printing is what was desired.

Reply:
You are correct - this isn't what we want since it would bypass `click` which, ultimately, should handle all of our output.

What we really want is to update the `get_messages_as_string` function.
https://github.com/DevOps-MBSE/AaC/blob/ebeb7c1831ac3586b0932a7ed30e2f061af14519/python/src/aac/plugins/validators/_validator_result.py#L30-L40

and add an `if`-check to ensure we only add the files that were processed if validation was successful but that class will already have all the data you need.

Reply:
Finally, I think it will be too noisy to print out every definition name in addition to every file name. We should limit the extra output to just filenames.

Reply:
Okay, I think I got this cleaned up. Please give it another look.